### ISSUE TITLE
The VSCode extension crashes

### ISSUE THREAD

**Issue by sasa1977** [elixir-lsp/elixir-ls#65]
I uninstalled the "official" ElixirLS and installed this fork from the marketplace. After restarting the code, I got a crash, and the extension is not working. The old ElixirLS worked just fine.

### Environment

* Elixir & Erlang versions (elixir --version): 1.9.1 && OTP 21
* Operating system: Ubuntu 19.04
* Client (editor or IDE): VSCode

### Crash report template

1.  _Create a new Mix project with `mix new empty`, then open that project with VS Code and open an Elixir file. Is your issue reproducible on the empty project? If not, please publish a repo on Github that does reproduce it._

The issue is not reproducible on an empty project. Unfortunately, I can't publish the repo in question.

2.  _Check the output log by opening `View > Output` and selecting "ElixirLS" in the dropdown. Please include any output that looks relevant. (If ElixirLS isn't in the dropdown, the server failed to launch.)_

```
** (UndefinedFunctionError) function ElixirLS.Utils.WireProtocol.intercept_output/2 is undefined (module ElixirLS.Utils.WireProtocol is not available)
    ElixirLS.Utils.WireProtocol.intercept_output(&ElixirLS.LanguageServer.JsonRpc.print/1, &ElixirLS.LanguageServer.JsonRpc.print_err/1)
    lib/language_server/cli.ex:6: ElixirLS.LanguageServer.CLI.main/0
    (stdlib) erl_eval.erl:680: :erl_eval.do_apply/6
    (elixir) lib/code.ex:240: Code.eval_string/3
[Error - 8:03:46 PM] Connection to server got closed. Server will not be restarted.
```

**Comment by axelson** [comment]
Thanks for the report. I get that error as well on Arch Linux. Seems like it is probably an issue with the packaging of the extension.

**Comment by WernerBuchert** [comment]
Exactly the same is on Windows 10 with VSCode 1.40.0.

Elixir 1.9.2
OTP 20.2
Hex 0.20.1

Unfortunately I haven't any output.

**Comment by dokie** [comment]
Same error on my MacBook Pro 

- Elixir 1.9.3
- OTP 21
- asdf 0.7.5

**Comment by axelson** [comment]
Okay, it looks like the problem is that the language server was compiled with OTP 22, which means that prior OTP versions are unable to load the `.beam` files. So as a work-around if you upgrade your project to erlang 22 everything should work. If you're using asdf and want to update to the latest versions of elixir and erlang that can be accomplished with:

```
asdf install elixir 1.9.4-otp-22
asdf install erlang 22.1.7
asdf local elixir 1.9.4-otp-22
asdf local erlang 22.1.7
```

Although any version of erlang `22.x` should work.

I am working on publishing an update to the extension in the marketplace but since someone else published the extension originally I am still working on getting access (getting the correct microsoft account has been a bit of a pain)

**Comment by WernerBuchert** [comment]
Ah okay, thanks. Yes, it work's fine on my Ubuntu 18.04 Laptop with Elixir 1.9.4 and OTP 22.1.6.
On the Windows PC I have to wait for a new version of Elixir.

**Comment by axelson** [comment]
Okay, I just published v0.2.27 that should fix this issue (although I just realized I forgot to mention it in the changelog when I published the extension)

**Comment by axelson** [comment]
Marking as closed but let us know if you are still running into this issue on version 0.2.27 (https://marketplace.visualstudio.com/items?itemName=elixir-lsp.elixir-ls) with an erlang version of 20 or higher.
